lib/models/tts.dart

```diff
@@ -42,4 +65,16 @@ class TtsModel {
       _tts.stop();
     }
   }
+
+  bool get isBotMuted {
+    return _isBotMuted;
+  }
+
+  set isBotMuted(bool value) {
+    _isBotMuted = value;
+    if (!value) {
```

Comment:
We don't need these here, these lines stop any active/pending TTS when TTS gets turned off.